Composer/packages/client/src/pages/knowledge-base/table-view.tsx

```diff
@@ -348,6 +349,7 @@ const TableView: React.FC<RouteComponentProps<{ dialogId: string; projectId: str
                         iconProps: { iconName: 'CodeEdit' },
                         onClick: () => {
                           navigateTo(`${baseURL}knowledge-base/${dialogId}/edit?C=${containerId}`);
+                          TelemetryClient.log('EditModeToggled');
```

Comment:
Also consider adding what context we are in (lg, lu, qna, etc)

Reply:
The current page is already added to every telemetry event.